src/_scss/core/_variables.scss

```diff
@@ -48,6 +48,7 @@ $color-black:                #000;
 $color-base:                 #555;
 $color-link:                 #0071bc;
 
+$color-gray-darker:          #1b1b1b;
```

Comment:
Oh sorry, there's already a #1b1b1b in _variables.scss.  I was suggesting using that variable.